docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -16,6 +16,8 @@ kubectl scale hazelcast hazelcast-sample --replicas 5
 
 == Upgrading
 
+Before upgrade, we recommend you to check link:https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rolling-upgrades#hazelcast-members-compatibility-guarantees[Hazelcast's Compatibility Guarantees].
```

Comment:
```suggestion
Before upgrading, review the Hazelcast xref:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[].
```

Reply:
Is it possible to reference to it like you did as `maintain-cluster:rolling-upgrades.adoc` is in another repository?: https://github.com/hazelcast/hz-docs

Reply:
Sorry, I didn't pick up on that. I believe the following should work:

xref:{page-latest-supported-hazelcast}@hazelcast:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[].